### PR TITLE
Add Info_dashboard package

### PR DESCRIPTION
**Pull request by tp82d1**
This package (`luci-app-tplink_dashboard`) introduces a dashboard interface for OpenWrt, inspired by the user interface of TP-Link firmware. The primary goal is to simulate the visual and functional aspects of TP-Link dashboards within the OpenWrt ecosystem.

#### **Features**

-   **TP-Link-Inspired Design**: A minimalist and visually appealing design that mirrors the layout and simplicity of TP-Link firmware dashboards.
-   **Dynamic Icons**: Includes interactive icons for Internet connection, device detail and connected devices.
    -   **Internet Status**: Displays the current connection status dynamically.
    -   **Device Detail**: Displays information about the router.

    -   **DHCP Monitoring**: Displays the number of active DHCP clients in real time.

#### **Testing**

The package was tested under the following conditions:

-   **OpenWrt Versions**:
    -   23.05.0

#### **Screenshots**
InternetStatus_NotConnected
dashboard
InternetStatus_connected
DeviceInfo
DHCP

### CODE REVIEW

**Comment by systemcrash** [comment]
Of course have written permission from them to use them, right?

**Comment by stangri** [comment]
First of all, I want to say it's a tremendous work to have a skin written for luci. If/when the IP/copyright issues of the assets and the name in the package title are resolved, I have the following comments on this PR:
1. you're including both the `applications/luci-mod-tplink_dashboard` and `luci-mod-tplink_dashboard` directories in your PR. I'm guessing it's an oversight and you only need one of them.
2. the proper location for modules (luci-mod-*) is actually `modules`, so I believe it should be `modules/luci-mod-tplink_dashboard`.
3. The iptables is not a default package for 23.05, 24.10 nor snapshots and you declare exec access to iptables/ip6tables binaries. I believe the mod should support nft natively before the PR is merged.

**Comment by systemcrash** [comment]
> Thanks for your replies, ill try my best to solve what you have mentioned in your replies. if I couldn't push them in same branch, i'll close and reopen new one :).

Please don't - figure out how to use git. That's what it's designed for.

**Comment by systemcrash** [comment]
Go grab some royalty-free vector graphics to replace the assets here. And credit/attribute any author involved.

**Comment by stangri** [comment]
> Go grab some royalty-free vector graphics to replace the assets here. And credit/attribute any author involved.

It's worth checking if TP-Link may have published those assets with the GPL code. 

Including their brand name in the name of the package might infringe on their copyright tho.

**Comment by systemcrash** [comment]
Looks considerably better. 
- can you replace the ❌ and ✅ with SVG versions?
- fix the test formalities, commit messages.

File: modules/luci-mod-info_dashboard/root/usr/share/luci/menu.d/luci-mod-tplink_dashboard.json
Comment (dannil) [inline modules/luci-mod-info_dashboard/root/usr/share/luci/menu.d/luci-mod-tplink_dashboard.json:1]: This file needs to be renamed to match `luci-mod-info_dashboard`.

File: modules/luci-mod-info_dashboard/root/usr/share/rpcd/acl.d/luci-mod-tplink_dashboard.json
Comment (dannil) [inline modules/luci-mod-info_dashboard/root/usr/share/rpcd/acl.d/luci-mod-tplink_dashboard.json:1]: This file needs to be renamed to match `luci-mod-info_dashboard`.

**Review by dannil — CHANGES_REQUESTED** [review]

**Pull request closed without merging**